recipe/meta.yaml

```diff
@@ -33,13 +33,10 @@ requirements:
     - pyyaml
   run_constrained:
     # modin
-    - pandas >=1.0.0,<3.0.0
```

Comment:
you can leave `pandas` for
1. `pandas` extra in https://github.com/snowflakedb/snowpark-python/blob/bb1ed3d5e618f3aa5744c7c5ca88b6b14bb4d5b2/setup.py#L112-L116
2. `snowflake-connector-python[pandas]`
as in https://github.com/snowflakedb/snowpark-python/blob/bb1ed3d5e618f3aa5744c7c5ca88b6b14bb4d5b2/setup.py#L33C7-L33C41
where there is the extra for `pandas` https://inspector.pypi.io/project/snowflake-connector-python/3.12.2/packages/5e/78/7e3ce98acef501ab3509958ed8f95e31720e225b7f478b7d7e70ff731fc9/snowflake_connector_python-3.12.2-cp39-cp39-manylinux_2_17_x86_64.manylinux2014_x86_64.whl/snowflake_connector_python-3.12.2.dist-info/METADATA#line.73